lib/google_scraping_web/controllers/error_handler.ex

```diff
@@ -0,0 +1,39 @@
+defmodule GoogleScrapingWeb.Controllers.ErrorHandler do
+  import Plug.Conn
+  import Phoenix.Controller
+
+  alias Ecto.Changeset
+  alias GoogleScrapingWeb.ErrorView
+
+  def render_error_json(conn, status),
+    do: render_error_json(conn, status, Phoenix.Naming.humanize(status))
+
+  def render_error_json(conn, status, message) do
+    conn
+    |> put_status(status)
+    |> put_view(ErrorView)
+    |> render("error.json", %{status: status, message: message})
+    |> halt()
+  end
+
+  def auth_error(conn, {type, _reason}, _opts) do
+    conn
+    |> put_status(:unauthorized)
+    |> put_view(ErrorView)
+    |> render("error.json", %{status: :unauthorized, message: type})
+  end
+
+  def build_changeset_error_message(changeset) do
+    errors =
+      Changeset.traverse_errors(changeset, fn {msg, opts} ->
+        Enum.reduce(opts, msg, fn {key, value}, acc ->
+          String.replace(acc, "%{#{key}}", to_string(value))
+        end)
+      end)
+
+    error_message =
```

Comment:
As we are not using this variable(`error_message `) anywhere, can we just return `Enum.map_join(errors, "\n", fn {key, errors} -> "#{key} #{Enum.join(errors, ", ")}" end)` directly

Reply:
Was fixed here: https://github.com/Nihisil/nimble-certification-elixir/pull/55/commits/ff6f9aeb36ecfcdb0b90e675fa6768de6f08a432